### DISCUSSION TITLE
Unable to remove collaboration from a repository shared by a friend

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54476] [Repositories]
### Select Topic Area

Question

### Body

Hi, I have an issue regarding the removal of my collaboration from a repository that was shared with me by a friend. Unfortunately, the repository owner is not responding and I don't have the necessary permissions to access the repository settings. I am unable to find the "Settings" button to remove my collaboration and have tried contacting the owner without success. Can you kindly assist me in resolving this issue? Thank you very much.

**Comment by GeoffreyFClark** [comment]
To remove yourself as a collaborator from a GitHub repo, go to the repo's main page and click the person icon below the "Watch" button. On the "Manage access" page, find your username and click "Leave."

**Reply by ghost** [reply]
Works! Thanks a lot for your help!!!

**Comment by herrye991** [comment]
You can leave collaboration at [Setting > Repository](https://github.com/settings/repositories) then click leave. good luck 😁

**Reply by ghost** [reply]
Works! Thanks a lot for your help!!!
